src/compat/eme/custom_media_keys/ie11_media_keys.ts

```diff
@@ -145,10 +146,23 @@ export default function getIE11MediaKeysCallbacks() {
     }
     return MSMediaKeysConstructor.isTypeSupported(keySystem);
   };
-  const createCustomMediaKeys = (keyType: string) => new IE11CustomMediaKeys(keyType);
+  const createCustomMediaKeys = (keyType: string) =>
+    new IE11CustomMediaKeys(keyType);
+  const setMediaKeys = (elt: HTMLMediaElement,
+                        mediaKeys: MediaKeys|ICustomMediaKeys|null): void => {
+    if (mediaKeys === null) {
+      return;
+    }
+    if (!isCustomMediaKeys(mediaKeys)) {
```

Comment:
There's something weird in checking if this is a `ICustomMediaKey` or not.

Don't we want to check specifically if it is a `IE11CustomMediaKeys` here?